### ISSUE TITLE
Cursor with green background instead of transparency on Linux inside Flatpak

### ISSUE THREAD

**Issue by Mailaender** [mriale/PyDPainter#98]
I was about to create a @flathub package but after testing I stumbled upon this:

![image](https://github.com/mriale/PyDPainter/assets/756669/0e86f7d1-52f2-4c4d-b348-451c5b31c31a)

https://github.com/Mailaender/flathub/blob/pydpainter/org.pydpainter.PyDPainter.yaml contains the versions I used.

**Comment by mriale** [comment]
@Mailaender Thanks for building this flatpack.

I can't reproduce this on Arch Linux. I set up a Python venv and installed these as in your flatpack:
* pygame 2.5.2
* numpy 1.25.2

The mouse cursor doesn't have a green background.

Does it do this when you run it normally outside the flatpack?

Are you running on an ARM Mac? Issue #78 fixed cursor rendering on an M1 Mac.

**Comment by Mailaender** [comment]
I can confirm that reverting https://github.com/mriale/PyDPainter/commit/ce69d4e3dd8cc91aa10d9ac800e7b61b7d83e7e7 fixes it. I am running on x64 Arch Linux.

**Comment by Mailaender** [comment]
I assume it has to do with the Python version in the [freedesktop-sdk](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/) or maybe I am missing an optional pygame dependency that you silently require here.

**Comment by mriale** [comment]
I compiled Python 3.12 from source and ran it in the venv with pygame 2.5.2 and numpy 1.25.2, and it still didn't show the green background for me. I'm pretty sure there are no other dependencies other than pygame and numpy.

I loaded your test flatpack and it seemed to function well.

I accepted your flatpack metatdata pull request.

I saw you included a patch to revert ce69d4e3dd8cc91aa10d9ac800e7b61b7d83e7e7. We'll have to make sure the patch is updated if the cursors ever need to change.

Thanks so much for enabling PyDPainter to be enjoyed by a much larger audience!
